week-3/E-array-map/exercise.js

```diff
@@ -4,11 +4,29 @@
 */
 
 // Complete this function -->
-function multiplyBy100(numbers) {}
+function multiplyBy100(numbers) {
+  return numbers.map(number => number * 100);
+}
+function multiplyBy100(numbers) {
+  return numbers.map(function times100(number) {
+    return number * 100;
+  });
+}
+function multiplyBy100(numbers) {
+  return numbers.map(function(number) {
+    return number * 100;
+  });
+}
+function multiplyBy100(numbers) {
+  return numbers.map(number => {
+    return number * 100;
+  });
+}
 
 // DEBUG
 // Debug your function by
 // - calling it with some sample input
 // - logging the output to the console
 
 module.exports = { multiplyBy100 };
+// This exercise i couldn't run and see the result on terminal
```

Comment:
Read the comment above under "DEBUG" - it tells you to add your own test data and log the output of calling your functions. The exercise files often contain instructions in the comments, so make sure you read them.